### PR TITLE
Feature/employment detail page

### PR DESCRIPTION
**Pull request by L1ght-time**

### CODE REVIEW

File: src/components/pages/EmploymentDetails/EmploymentDetailCompany/EmploymentDetailCompany.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { CardTitle } from '@/components/shared/Card';
+import { ICompany } from '@/types/IEmploymentHistory';
+
+export const EmploymentDetailCompany: FC<Omit<ICompany, 'image'>> = ({ title }) => (
```
Comment (mykyta-lazurenko) [inline src/components/pages/EmploymentDetails/EmploymentDetailCompany/EmploymentDetailCompany.tsx:7]: Why `Omit` image?

File: src/components/pages/EmploymentDetails/EmploymentDetailHead/EmploymentDetailHead.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
+import Image from 'next/image';
+import { MdOutlineKeyboardArrowLeft } from 'react-icons/md';
+import { useRouter } from 'next/router';
+
+import { ICompany } from '@/types/IEmploymentHistory';
+
+export const EmploymentDetailHead: FC<Omit<ICompany, 'title'>> = ({ image }) => {
```
Comment (mykyta-lazurenko) [inline src/components/pages/EmploymentDetails/EmploymentDetailHead/EmploymentDetailHead.tsx:8]: Why `Omit` title?

File: src/components/pages/EmploymentDetails/EmploymentDetails.tsx [hunk 3]
```diff
@@ -24,5 +31,22 @@ export const EmploymentDetails: FC = () => {
...
+  } = employmentHistory;
+
+  return (
+    <Card className="w-1/2 h-full p-5 gap-2">
+      <EmploymentDetailHead image={image} />
+      <EmploymentDetailCompany title={title} />
+      <EmploymentDetailPosition position={position} />
+      <EmploymentDetailPeriod {...period} />
+      <EmploymentDetailResponsibilities responsibilities={responsibilities} />
+      <EmploymentDetailTechnologyStack technologyStack={technologyStack} />
```
Comment (mykyta-lazurenko) [inline src/components/pages/EmploymentDetails/EmploymentDetails.tsx:49]: Better remove these components and paste content here

File: src/helpers/getDiffPeriodOfDate.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import { IPeriod } from '@/types/IEmploymentHistory';
+
+export const getDiffPeriodOfDate = ({ start, end }: IPeriod) => {
+  const { years, months } = DateTime.fromISO(end)
+    .diff(DateTime.fromISO(start), ['years', 'months'])
+    .plus({ month: 1 })
+    .normalize();
+
+  return { years, months };
+};
```
Comment (mykyta-lazurenko) [inline src/helpers/getDiffPeriodOfDate.ts:12]:
```suggestion
export const getDiffPeriodOfDate = ({ start, end }: IPeriod) => DateTime.fromISO(end)
    .diff(DateTime.fromISO(start), ['years', 'months'])
    .plus({ month: 1 })
    .normalize();
```

**Review by mykyta-lazurenko — APPROVED** [review]

File: src/components/pages/Office/Office.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { FC } from 'react';
+import { useRouter } from 'next/router';
+
+export const Office: FC = () => {
+  const {
+    query: { office },
+  } = useRouter();
+
+  console.log({ office });
```
Comment (asd-ez) [inline src/components/pages/Office/Office.tsx:9]: Remove logs
```suggestion
```

File: src/components/pages/Office/OfficeInfo/OfficeInfo.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+import { fetcher } from '@/constants';
+import { Loader } from '@/components/shared/Loader';
+
+export const OfficeInfo: FC = () => {
+  const {
+    query: { officeInfoId },
+  } = useRouter();
+
+  const { data, isLoading } = useSWR(officeInfoId ? `/api/officeInfo/${officeInfoId}` : null, fetcher);
```
Comment (asd-ez) [inline src/components/pages/Office/OfficeInfo/OfficeInfo.tsx:13]: Need more variable understanding of naming, here you have office info, `data` it's bad naming, e.g.:
```suggestion
  const { data: officeInfo, isLoading } = useSWR(officeInfoId ? `/api/officeInfo/${officeInfoId}` : null, fetcher);
```

**Review by asd-ez — APPROVED** [review]

**Pull request merged**